src/backend/distributed/executor/citus_custom_scan.c

```diff
@@ -221,20 +212,7 @@ RouterCreateScan(CustomScan *scan)
 	{
 		Assert(isModificationQuery);
 
-		if (IsMultiRowInsert(workerJob->jobQuery) ||
-			MultiShardConnectionType == SEQUENTIAL_CONNECTION)
-		{
-			/*
-			 * Multi shard modifications while multi_shard_modify_mode equals
-			 * to 'sequential' or Multi-row INSERT are executed sequentially
-			 * instead of using parallel connections.
-			 */
-			scanState->customScanState.methods = &RouterSequentialModifyCustomExecMethods;
-		}
-		else
-		{
-			scanState->customScanState.methods = &RouterMultiModifyCustomExecMethods;
-		}
+		scanState->customScanState.methods = &RouterModifyCustomExecMethods;
```

Comment:
I think the distinction between this block and the one above is no longer relevant.

Reply:
makes sense